deploy/template.yaml

```diff
@@ -1203,22 +1255,31 @@ Resources:
             Action:
               - kms:*
             Resource: "*"
+
+  S3KmsKey:
+    Type: AWS::KMS::Key
+    Properties:
+      EnableKeyRotation: true
+      KeyPolicy:
+        Version: 2012-10-17
+        Statement:
           - Effect: Allow
             Principal:
-              Service: "dynamodb.amazonaws.com"
+              AWS: !Sub "arn:aws:iam::${AWS::AccountId}:root"
+            Action:
```

Comment:
I think the owning account probably needs `kms:*` otherwise CloudFormation won't be able to manage the key.